deps/rabbitmq_cli/mix.exs

```diff
@@ -149,19 +149,15 @@ defmodule RabbitMQCtl.MixfileBase do
           end
         end
 
-        # We disable compilation for rabbit_common and amqp_client
-        # because Erlang.mk already built them.
         [
           {
             :rabbit_common,
             path: Path.join(deps_dir, "rabbit_common"),
-            compile: false,
             override: true
           },
           {
             :amqp_client,
             path: Path.join(deps_dir, "amqp_client"),
-            compile: false,
```

Comment:
@lukebakken Given the changes to `deps/rabbitmq_cli/rabbitmqctl.bzl` in #4028, I suspect it is no longer necessary to remove these lines.